components/style/properties/helpers/animated_properties.mako.rs

```diff
@@ -34,10 +34,12 @@ use super::ComputedValues;
 use values::Auto;
 use values::{CSSFloat, CustomIdent, Either};
 use values::animated::ToAnimatedValue;
+#[cfg(feature = "gecko")] use values::animated::AnimatedStrokeDashArrayList;
 use values::animated::effects::BoxShadowList as AnimatedBoxShadowList;
 use values::animated::effects::Filter as AnimatedFilter;
 use values::animated::effects::FilterList as AnimatedFilterList;
 use values::animated::effects::TextShadowList as AnimatedTextShadowList;
+#[cfg(feature = "gecko")] use values::computed::LengthOrPercentage as AnimatedLengthOrPercentageOrNumber;
```

Comment:
Hm, I think renaming this as `AnimatedLengthOrPercentageOrNumber` can cause confusions since `LengthOrPercentage` doesn't contain Number. Also we may need to rename it in the future if some other property needs a `LengthOrPercentage` animation type.